pkg/apis/networking/validation/validation_test.go

```diff
@@ -1061,6 +1224,24 @@ func TestValidateIngressTLS(t *testing.T) {
 			}
 		}
 	}
+
```

Comment:
Will remove this and put in a different commit since it is part of wildcard hostnames and not specific to PathType